### ISSUE TITLE
TestTraffic/virtualservice/shifting-80 flakes

### ISSUE THREAD

**Issue by hzxuzhonghu** [istio/istio#32263]
https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/32254/integ-pilot-multicluster-tests_istio/1383959079608651776

@howardjohn Do you have any clue which pr causes this

**Comment by howardjohn** [comment]
Not exactly, there is some more info in https://github.com/istio/istio/issues/32208 though. Lets discuss there
